client/components/plans/plan-list/index.jsx

```diff
@@ -25,32 +26,40 @@ module.exports = React.createClass( {
 	},
 
 	render: function() {
-		var plans = this.props.plans,
-			showJetpackPlans = false,
+		var className = '',
+			plans = this.props.plans,
+			hideFreePlan = this.props.hideFreePlan ? true : false,
 			isLoadingSitePlans = ! this.props.isInSignup && ! this.props.sitePlans.hasLoadedFromServer,
+			numberOfPlaceholders = 3,
 			site,
 			plansList,
 			currentPlan;
 
+		if ( this.props.sites ) {
```

Comment:
I don't think this is a problem because <a href="https://github.com/Automattic/wp-calypso/blob/b894983aee8e51df16c6fa944dc487dd7cc489aa/client/signup/steps/plans/index.jsx#L91">we don't pass a sites prop in the signup step</a>.